Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -16,16 +16,11 @@
     - Single
     soundGunshot:
       path: /Audio/Weapons/Guns/Gunshots/laser.ogg
+  - type: PowerCellSlot
     cellSlot:
       ejectOnUse: true
-      insertSound: /Audio/Weapons/Guns/MagIn/revolver_magin.ogg
-      ejectSound: /Audio/Weapons/Guns/MagOut/revolver_magout.ogg
       soundOptions: 
         volume: -2
```

Comment:
why remove the insert/eject sounds?

Reply:
They got set as defaults in the C# file.